docs/docs/how-to/images-and-media/using-gatsby-plugin-image.md

```diff
@@ -233,24 +233,24 @@ module.exports = {
 
 ## Using images from a CMS or CDN
 
-Many source plugins have native support for `gatsby-plugin-image`, with images served directly from a content delivery network (CDN). This means that builds are faster, because there is no need download images and process them locally. The query syntax varies according to the plugin, as do the supported transformation features and image formats. Make sure you update to the latest version of the source plugin to ensure there is support. For plugins that are not in this list you can use [dynamic images from `gatsby-transformer-sharp`](#dynamic-images).
+Many source plugins have native support for `gatsby-plugin-image`, with images served directly from a content delivery network (CDN). This means that builds are faster, because there is no need to download images and process them locally. The query syntax varies according to the plugin, as do the supported transformation features and image formats. Make sure you update to the latest version of the source plugin to ensure there is support. For plugins that are not in this list you can use [dynamic images from `gatsby-transformer-sharp`](#dynamic-images).
 
 ### Source plugins
 
 These source plugins support using `gatsby-plugin-image` with images served from their CDN.
 
 - [AgilityCMS](https://github.com/agility/gatsby-image-agilitycms)
-- [Contentful](/plugins/gatsby-source-contentful/#using-the-new-gatsby-image-plugin)
-- [DatoCMS](/plugins/gatsby-source-datocms/#integration-with-gatsby-image)
-- [GraphCMS](/plugins/gatsby-source-graphcms/#usage-with-gatsby-plugin-image)
-- [Sanity](/plugins/gatsby-source-sanity/#using-images)
+- [Contentful](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-contentful#using-the-new-gatsby-image-plugin)
+- [DatoCMS](https://github.com/datocms/gatsby-source-datocms#integration-with-gatsby-image)
+- [GraphCMS](https://github.com/GraphCMS/gatsby-source-graphcms#usage-with-gatsby-plugin-image)
+- [Sanity](https://github.com/sanity-io/gatsby-source-sanity#using-images)
 - [Shopify](https://github.com/gatsbyjs/gatsby-source-shopify-experimental#images)
```

Comment:
Can you please keep them as is, all relative links are going to https://www.gatsbyjs.com

Reply:
I've reverted those to relative. Intention was unclear due to the Shopify CMS and Agility CMS using non-relative links.

Reply:
Have also reverted the imgix one changed.